modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,98 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies implement an additional level of granularity, where access is controlled based on the a criteria, called resource-level policies. 
```

Comment:
Maybe we should say before that by default policies are applied to the entire API unless more granularity is needed?

Reply:
Done! Thanks!